practice-01/js/index.js

```diff
@@ -56,8 +80,24 @@ function checkEmailError(input) {
     }
     return isEmailError;
 }
+
+/**
+ * Display an error message when the user enters the wrong email
+ * @param isEmailError : True || False
+ * @param input : Element of email ID
+ */
+function validationEmail(isEmailError, input) {
+    if (isEmailError == false) {
+        showError(input, 'Email Invalid');
+    } else {
+        showSuccess(input);
+    }
+}
+
 /**
- * Validation for charcaters
+ * Check special characters
+ * @param input : Element of the argument
+ * @returns {boolean}

```

Comment:
Example: return true if text string match and pass the regex rule. Otherwise, return false.